_tools/cite-guide.md

```diff
@@ -0,0 +1,459 @@
+| 2.16.840.1.101.3.2.1.48.109 | id-fpki-common-pivAuth-derived | 2.16.840.1.101.3.2.1.3.40 |
+| 2.16.840.1.101.3.2.1.48.110 | id-fpki-common-pivAuth-derived-hardware | 2.16.840.1.101.3.2.1.3.41 |
+
+### Federal Agency PKI Test OIDs
+
+#### Department of Defense (DoD)
+
+| Test OID                    | Policy                 | Production OID     |
+| --------------------------- | ---------------------- | ------------------ |
+| 2.16.840.1.101.3.2.1.48.68 | id-US-dod-basic | 2.16.840.1.101.2.1.11.2 |
+| 2.16.840.1.101.3.2.1.48.69 | id-US-dod-medium-2048 | 2.16.840.1.101.2.1.11.18 |
+| 2.16.840.1.101.3.2.1.48.70 | id-US-dod-mediumhardware-2048 | 2.16.840.1.101.2.1.11.19 |
+| 2.16.840.1.101.3.2.1.48.71 | id-US-dod-FORTEZZA | 2.16.840.1.101.2.1.11.4 |
+| 2.16.840.1.101.3.2.1.48.72 | id-US-dod-type1 | 2.16.840.1.101.2.1.11.6 |
+| N/A | id-US-dod-mediumNPE-2048 | 2.16.840.1.101.2.1.11.17 |
+| N/A | id-US-dod-mediumNPE-112 | 2.16.840.1.101.2.1.11.36 |
+| N/A | id-US-dod-mediumNPE-128 | 2.16.840.1.101.2.1.11.37 |
+| N/A | id-US-dod-mediumNPE-192 | 2.16.840.1.101.2.1.11.38 |
+| N/A | id-US-dod-medium-112 | 2.16.840.1.101.2.1.11.39 |
+| N/A | id-US-dod-medium-128 | 2.16.840.1.101.2.1.11.40 |
+| N/A | id-US-dod-medium-192 | 2.16.840.1.101.2.1.11.41 |
+| N/A | id-US-dod-mediumHardware-112 | 2.16.840.1.101.2.1.11.42 |
+| N/A | id-US-dod-mediumHardware-128 | 2.16.840.1.101.2.1.11.43 |
+| N/A | id-US-dod-mediumHardware-192 | 2.16.840.1.101.2.1.11.44 |
+
+#### Department of Defense External CA (DoD ECA)
+
+| Test OID                    | Policy                 | Production OID     |
+| --------------------------- | ---------------------- | ------------------ |
+| N/A | ECA medium | 2.16.840.1.101.3.2.1.12.1 |
+| N/A | ECA Medium Hardware | 2.16.840.1.101.3.2.1.12.2 |
+| N/A | ECA Medium Token | 2.16.840.1.101.3.2.1.12.3 |
+
+#### Department of Energy (DOE)
+
+| Test OID                    | Policy                 | Production OID     |
+| --------------------------- | ---------------------- | ------------------ |
+| 2.16.840.1.101.3.2.1.48.19 | doe-basic | 2.16.840.1.101.3.2.1.10.1 |
+| 2.16.840.1.101.3.2.1.48.20 | doe-medium | 2.16.840.1.101.3.2.1.10.2 |
+| 2.16.840.1.101.3.2.1.48.21 | doe-high | 2.16.840.1.101.3.2.1.10.3 |
+| 2.16.840.1.101.3.2.1.48.22 | doe-medium-v2 | 2.16.840.1.101.3.2.1.10.4 |
+
+#### Department of Homeland Security (DHS)
+
+| Test OID                    | Policy                 | Production OID     |
+| --------------------------- | ---------------------- | ------------------ |
+| 2.16.840.1.101.3.2.1.15.31 | id-dhs-certpcy-rudimentary | 2.16.840.1.101.3.2.1.15.1 |
+| 2.16.840.1.101.3.2.1.15.32 | id-dhs-certpcy-basic | 2.16.840.1.101.3.2.1.15.2 |
+| 2.16.840.1.101.3.2.1.15.33 | id-dhs-certpcy-medium | 2.16.840.1.101.3.2.1.15.3 |
+| 2.16.840.1.101.3.2.1.15.34 | id-dhs-certpcy-high | 2.16.840.1.101.3.2.1.15.4 |
+| 2.16.840.1.101.3.2.1.15.35 | id-dhs-certpcy-mediumHardware | 2.16.840.1.101.3.2.1.15.5 |
+
+#### Department of Justice (DOJ)

```

Comment:
Justice OIDs should no longer be active?  No cross-certs in prod.

Reply:
Done